components/pdf_preview.jsx

```diff
@@ -40,6 +40,8 @@ export default class PDFPreview extends React.PureComponent {
             loading: true,
             success: false,
         };
+
+        this.pdfCanvas = React.createRef();
```

Comment:
Hey @ArturBa Can you change this to be `this.pdfCanvasRef` for consistency with other instances?

Reply:
Also remove `/* eslint-disable react/no-string-refs */` instances from the file

Reply:
done